### PR TITLE
Remove Target Path From Publish Package Dialog

### PR DESCRIPTION
**Pull request by OliverEGreen**
### Purpose

Cherrypicked from: #12422. This PR removes the Target Path column in the `PublishPackageView` and its associate properties in the `PackageItemRootViewModel`.

SbA58DVHmV

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Removes a redundant column from the Publish Package dialog.

### Reviewers

@QilongTang @mjkkirschner 

### FYIs

@Amoursol
@SHKnudsen

### CODE REVIEW

**Review by QilongTang — APPROVED** [review]

File: src/DynamoCoreWpf/Views/PackageManager/PublishPackageView.xaml [hunk 1]
```diff
@@ -695,23 +695,18 @@
                                                 MaxWidth="85"
                                                 Binding="{Binding Path=IsSelected, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
                                                 ElementStyle="{StaticResource CheckBoxStyle}"
-                                                Header="Node Library"
+                                                Header="{x:Static p:Resources.PublishPackageViewNodeLibrary}"
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/PackageManager/PublishPackageView.xaml:698]: Nice catch

**Comment by reddyashish** [comment]
Will need to resolve the conflicts here.

**Comment by QilongTang** [comment]
> Will need to resolve the conflicts here.

I just did, let's wait for the checks again

**Comment by QilongTang** [comment]
All tests pass, merging

**Pull request merged**